sdk/enter.go

```diff
@@ -197,29 +198,32 @@ func (e *enter) MountGnupgHome() error {
 	// Additionally, set the GNUPGHOME variable so commands run with sudo
 	// can also use it.
 	newHome := filepath.Join(e.Chroot, e.User.HomeDir, ".gnupg")
-	if err := os.Mkdir(newHome, 0700); err != nil {
+	if err := os.Mkdir(newHome, 0700); err != nil && !os.IsExist(err) {
```

Comment:
Should the changes in this commit to `MountGnupugHome` be in the previous commit?